src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -642,7 +642,7 @@ private function addService($id, $definition)
      *$lazyInitializationDoc
      * $return
      */
-    {$visibility} function {$methodName}($lazyInitialization)
+    protected function {$methodName}($args)
```

Comment:
imo you should not change this method visibility in this PR as this is what breaks the tests and due to how the tests are executed they won't pass until the fix is merged, so we could forget bugs...